api/src/main/java/com/epam/esm/api/converter/SortRequestDtoToModelConverter.java

```diff
@@ -0,0 +1,44 @@
+package com.epam.esm.api.converter;
+
+import com.epam.esm.api.dto.SortRequestDto;
+import com.epam.esm.model.Sort;
+import com.epam.esm.model.SortRequest;
+import org.modelmapper.AbstractConverter;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+@Component
+public class SortRequestDtoToModelConverter extends AbstractConverter<SortRequestDto, SortRequest> implements Converter<SortRequestDto, SortRequest> {
+
+    private static final Sort.SortOrder DEFAULT_SORT_DIRECTION = Sort.SortOrder.ASC;
+
+    private static final String SPLITERATOR = ",";
+    private static final int LENGTH_SORT_FIELD_ONLY = 1;
+    private static final int INDEX_SORT_FIELD = 0;
+    private static final int INDEX_SORT_DIRECTION = 1;
+
+    @Override
+    public SortRequest convert(SortRequestDto sortRequestDto) {
+        String sortDto = sortRequestDto.getSort();
+        Sort sort = sortDto == null
+                ? null
+                : this.convert(sortDto);
+        String thenSortDto = sortRequestDto.getThenSort();
+        Sort thenSort = thenSortDto == null
+                ? null
+                : this.convert(thenSortDto);
+        return new SortRequest(sort, thenSort);
+    }
+
+    private Sort convert(String sortDto) {
+        String[] parameters = sortDto.split(SPLITERATOR);
+        String field = parameters[INDEX_SORT_FIELD];
```

Comment:
I would suggest the structure below:
SortDto with 2 fields: sortBy, sortOrder. 
SortListDto with 1 field: List<SortDto> 

This  is more extensible (f.e. user would like to sort by 3 fields). Also this allow to get rid of string splitting.